docs/layouts/shortcodes/guides/signup.html

```diff
@@ -1,16 +1,31 @@
 <p>Depending on whether or not you have an existing project you want to migrate to Platform.sh there are three ways to get started.</p>
 
-<h3>With a template</h3>
+<h3>From a third party Git provider</h3>
```

Comment:
I'm not convinced about making this the first option, because if you haven't setup the config files you'll get an error immediately upon adding the integration.  That is not a good user experience.  Also, we deliberately are trying to balance new and migration users in these guides.  Shifting it entirely to a migration-centric focus is a change of direction I'm not comfortable with at all.

Reply:
That's a fair point

Reply:
Reverted this file back to original order